boards/nordic/nrf54lv10dk/nrf54lv10dk_nrf54lv10a_cpuapp_ns.dts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/dts-v1/;
+
+#define USE_NON_SECURE_ADDRESS_MAP 1
+
+#include "nrf54lv10a_cpuapp_common.dtsi"
+
+/ {
+	compatible = "nordic,nrf54lv10dk_nrf54lv10a-cpuapp";
+	model = "Nordic nRF54LV10 DK nRF54LV10A Application MCU";
+
+	chosen {
+		zephyr,code-partition = &slot0_partition;
+		zephyr,sram = &cpuapp_sram;
+		zephyr,bt-hci = &bt_hci_sdc;
+		zephyr,entropy = &psa_rng;
+
+	};
+
+	/delete-node/ rng;
+
+	psa_rng: psa-rng {
+		status = "okay";
+	};
+};
+
+/ {
+	/*
+	 * Default SRAM planning when building for nRF54LV10 with ARM TrustZone-M support.
+	 * - Lowest 72 kB SRAM allocated to Secure image (sram0_s).
+	 * - Upper 72 kB SRAM allocated to Non-Secure image (sram0_ns).
+	 *
+	 * nRF54LV10 has 192 kB of volatile memory (SRAM) but the last 42kB are reserved for
+	 * the FLPR MCU.
+	 * This static layout needs to be the same with the upstream TF-M layout in the
+	 * header flash_layout.h of the relevant platform. Any updates in the layout
+	 * needs to happen both in the flash_layout.h and in this file at the same time.
+	 */
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		sram0_s: image_s@20000000 {
+			/* Secure image memory */
+			reg = <0x20000000 DT_SIZE_K(72)>;
+		};
+
+		sram0_ns: image_ns@20012000 {
+			/* Non-Secure image memory */
+			reg = <0x20012000 DT_SIZE_K(72)>;
+		};
+	};
+};
+
+&cpuapp_rram {
+	compatible = "soc-nv-flash";
+	erase-block-size = <4096>;
+	write-block-size = <16>;
```

Comment:
why are these here? Should be in soc dts file

Reply:
Sure